canonical-web-and-design/tools.demos.haus
# Add download links for banners

## Done

- Moved form into a modal
- Added an index of all banners
- Added back to top links under each banner
- Added a download all banners button
- Added a download button for each banner

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8057/?title=Ubuntu+for+the+internet+of+things&subtitle=From+smart+homes+to+smart+drones%2C+robots%2C+and+industrial+systems%2C+Ubuntu+is+the+new+standard+for+embedded+Linux.&illustration_url=https%3A%2F%2Fassets.ubuntu.com%2Fv1%2Fa70bfab2-Managed-apps_wht.svg&background=grad#banner-form-modal
- Check that the "Build banners" button triggers a modal with the form (and that it works)
- Check that the index links go to the right banner
- Check that the back to top links work
- Check that the "Download all banners" button downloads all banners
- Check that each banners download button downloads the right banner and that the file is named appropriately

## Issue / Card

Fixes #8
